docs/tests/code-coverage.md

```diff
@@ -28,6 +28,10 @@ Code Climate metrics is a good place to start. We're happy to see PRs that
 refactor existing code to reduce duplication and increase readability, given
 that those refactors are well tested.
 
+## Codecov
+
+We use Codecov to track code coverage. Codecov leaves a comment PR's that indicate the a percentage change in code coverage compared to main. We encourage you to keep the code coverage percentage positive.
```

Comment:
Are we trying to keep it ~ 80% coverage? Perhaps 80% feels arbitrary...but I've also heard this number as a general guideline before -- I should do some research on it though... 

What are your thoughts about the percentage value as a soft guardrail ?

Reply:
I think 80% is reasonable since we're at 81% at the moment. I'll add this and hoping to get what other folks thoughts who are reviewing the PR too.

Reply:
@maestromac perhaps this percentage is something we could as a check to our CI?

Reply:
Our internal goal is to keep it 99% but I think that the more acceptable goal for the tool is to never lower the coverage.

> @maestromac perhaps this percentage is something we could as a check to our CI?

yes Codecov should have that 'check' that we can implement